component/component/vshn_redis.jsonnet

```diff
@@ -94,6 +94,25 @@ local restoreRole = kube.ClusterRole(restoreRoleName) {
   ],
 };
 
+local helmMonitoringClusterRole = kube.ClusterRole('allow-helm-monitoring-resources') {
+  rules: [
+    {
+      apiGroups: [ 'monitoring.coreos.com' ],
+      resources: [ 'servicemonitors' ],
+      verbs: [ '*' ],
```

Comment:
I was thinking about this configuration, are we sure we need all verbs? Maybe we need only `list, watch and get` only.

Reply:
it also requires create, update and delete, therefore I used `'*'` asteriks value - easier to read :)